src/components/advanced-color-picker/advanced-color-picker.pw.tsx

```diff
@@ -0,0 +1,421 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import { AdvancedColorPickerCustom } from "./components.test-pw";
+import {
+  simpleColorPicker,
+  currentColorDescription,
+  advancedColorPickerCell,
+  simpleColorPickerInput,
+  simpleColorPickerComponent,
+  advancedColorPickerPreview,
+} from "../../../playwright/components/advanced-color-picker";
+import { alertDialogPreview as advancedColorPickerParent } from "../../../playwright/components/dialog/index";
+import { closeIconButton } from "../../../playwright/components/index";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import { checkAccessibility } from "../../../playwright/support/helper";
+import { HooksConfig } from "../../../playwright";
+
+test.describe("when focused", () => {
+  test("should have the expected styling when the focusRedesignOptOut is false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<AdvancedColorPickerCustom open />);
+
+    const icon = closeIconButton(page);
+    await icon.focus();
+    await expect(closeIconButton(page)).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+    await expect(closeIconButton(page)).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+  });
+
+  test("should have the expected styling when the focusRedesignOptOut is true", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<AdvancedColorPickerCustom open />, {
+      hooksConfig: {
+        focusRedesignOptOut: true,
+      },
+    });
+
+    const icon = closeIconButton(page);
+    await icon.focus();
+    await expect(closeIconButton(page)).toHaveCSS(
+      "outline",
+      "rgb(255, 188, 25) solid 3px"
+    );
+  });
+});
+
+test.describe(
+  "should render AdvancedColorPicker component and check functionality",
+  () => {
+    ([
+      ["ArrowUp", 2],
+      ["ArrowLeft", 6],
+      ["ArrowRight", 8],
+    ] as [string, number][]).forEach(([key, index]) => {
+      test(`should use ${key} key and move selection`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<AdvancedColorPickerCustom />);
+
+        const picker = simpleColorPicker(page, 7);
+        await picker.press(key);
+        await expect(simpleColorPickerInput(page, index)).toHaveAttribute(
+          "aria-checked",
+          "true"
+        );
+      });
+    });
+
+    test(`should use key and move selection`, async ({ mount, page }) => {
```

Comment:
Shouldn't this test be included in the test line 56 - 77 above?

Reply:
Correct, that test was just a simplified version of the previous one added when I was trying to make it work. It should have been removed and is now.